dotnet/src/SemanticKernel.Abstractions/Contents/FunctionCallContentBuilder.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// A builder class for creating <see cref="FunctionCallContent"/> objects from incremental function call updates represented by <see cref="StreamingFunctionCallUpdateContent"/>.
+/// </summary>
+[Experimental("SKEXP0001")]
+public sealed class FunctionCallContentBuilder
```

Comment:
Same here, we should add an example that shows how to use this class.

Reply:
Added, please check - OpenAI_FunctionCalling.RunStreamingChatAPIWithManualFunctionCallingAsync.